### PR TITLE
Migration: SMAIndicator

### PR DESCRIPTION
**Pull request by marvin19**
Improved SMAIndicator.

### CODE REVIEW

**Review by bre1470 — CHANGES_REQUESTED** [review]
Thank you. Some requests:
- [ ] `/* eslint-disable/enable valid-jsdoc */` are redundant and can be removed. The rule has been relaxed and downgraded to a warning.
- [ ] Indicator options lacks the last section comment.

File: ts/Stock/Indicators/IKH/IKHIndicator.ts [hunk 1]
```diff
@@ -238,6 +238,10 @@ class IKHIndicator extends SMAIndicator {
      * @requires     stock/indicators/ichimoku-kinko-hyo
      * @optionparent plotOptions.ikh
      */
+
+    /**
+     * Default options
+     */
```
Comment (bre1470) [inline ts/Stock/Indicators/IKH/IKHIndicator.ts:244]: This comment should be removed. We already have the comment above this one.

File: ts/Stock/Indicators/SMA/SMAIndicator.ts [hunk 2]
```diff
@@ -104,6 +104,10 @@ class SMAIndicator extends LineSeries {
      * @requires     stock/indicators/indicators
      * @optionparent plotOptions.sma
      */
+
+    /**
+     * Default options
+     */
```
Comment (bre1470) [inline ts/Stock/Indicators/SMA/SMAIndicator.ts:110]: Same here.

**Review by bre1470 — CHANGES_REQUESTED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**